torch/autograd/function.py

```diff
@@ -7,16 +7,16 @@
 import warnings
 from collections import OrderedDict
 from typing import Any, List, Optional
+import torch.autograd.forward_ad as fwAD
 
 # Formerly known as: _ContextMethodMixin
 class FunctionCtx(object):
 
     def save_for_backward(self, *tensors: torch.Tensor):
         r"""Saves given tensors for a future call to :func:`~Function.backward`.
 
-        **This should be called at most once, and only from inside the**
-        :func:`forward` **method. This should only be called with input or
-        output tensors**
+        This should be called at most once, and only from inside the
+        :func:`forward` method. This method should only be called with tensors.
```

Comment:
Not sure we want to modify this. We don't want the user to use it for non-input/output. And it does say "should" hinting at the fact that it is not checked :P